MapboxNavigation/RouteMapViewController.swift

```diff
@@ -706,7 +714,6 @@ extension RouteMapViewController: NavigationViewDelegate {
                 guard line.pointCount > 0 else { continue }
                 let featureCoordinates =  Array(UnsafeBufferPointer(start: line.coordinates, count: Int(line.pointCount)))
                 let featurePolyline = LineString(featureCoordinates)
-                let slicedLine = stepShape.sliced(from: closestCoordinate)!
```

Comment:
Thanks, moving the slicing out of the loop should have a measurable impact on areas where there are lots of overlapping lines.